.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
 .vscode
 /vendor
-lcov.info
\ No newline at end of file
+lcov.info
+Cargo.lock
```

Comment:
Thanks, this shouldn't have been in here in the first place. 👍🏻

Reply:
I misread this diff, I thought it was removing Cargo.lock, but on relook I see it is adding it. We don't want the Cargo.lock file to be excluded from git, for the reasons described here: https://github.com/stellar/rs-soroban-env/pull/1351#issuecomment-1951513131